README.md

```diff
@@ -124,17 +126,19 @@ case fetch_user(id) do
 end
 ```
 
-### Similar Libraries
-
-For reference.
-
-- [exceptional](https://github.com/expede/exceptional)
-- [elixir-monad](https://github.com/nickmeharry/elixir-monad)
-- [happy_with](https://github.com/vic/happy_with)
-- [monad](https://github.com/rmies/monad)
-- [ok_jose](https://github.com/vic/ok_jose)
-- [towel](https://github.com/knrz/towel)
-
-*Possible extensions to include implementing bind on structs so that errors can be better handled.
-Implement a catch functionality for functions that error.
-Implement existing monad library protocols so can extend similar DB functionality e.g. slick*
+## External Links and Resources
+
```

Comment:
I preferred links to forum in the relevant sections and think links to hex docs should stay at the top.

The reason I wanted to move more to module docs was to make the README an overview that quickly lead you to the docs if you were serious about using it